pandas/_libs/interval.pyx

```diff
@@ -81,26 +81,45 @@ cdef class Interval(IntervalMixin):
 
     Parameters
     ----------
-    left : value
-        Left bound for the interval
-    right : value
-        Right bound for the interval
+    left : orderable object
+        Left bound for the interval.
+    right : orderable object
+        Right bound for the interval.
     closed : {'left', 'right', 'both', 'neither'}, default 'right'
         Whether the interval is closed on the left-side, right-side, both or
-        neither
+        neither.
+    
+    Notes
+    -----
+    You must be able to compare the parameters **left** and **right**,
```

Comment:
Can you use single backticks to refer to the keywords? (like `` `left` ``), and double backticks for the code snippet on the line below ?